appstore/model.go

```diff
@@ -66,22 +84,37 @@ type (
 		BundleID                   string  `json:"bundle_id"`
 		ApplicationVersion         string  `json:"application_version"`
 		DownloadID                 int64   `json:"download_id"`
+		VersionExternalIdentifier  int64   `json:"version_external_identifier"`
 		OriginalApplicationVersion string  `json:"original_application_version"`
 		InApp                      []InApp `json:"in_app"`
+		ReceiptCreationDate
 		RequestDate
 		OriginalPurchaseDate
 	}
 
+	// A pending renewal may refer to a renewal that is scheduled in the future or a renewal that failed in the past for some reason.
+	PendingRenewalInfo struct {
+		SubscriptionExpirationIntent   string `json:"expiration_intent"`
+		SubscriptionAutoRenewProductID string `json:"auto_renew_product_id"`
+		SubscriptionRetryFlag          string `json:"is_in_billing_retry_period"`
+		SubscriptionAutoRenewStatus    string `json:"auto_renew_status"`
+		SubscriptionPriceConsentStatus string `json:"price_consent_status"`
+		ProductID                      string `json:"product_id"`
+	}
+
 	// The IAPResponse type has the response properties
 	// We defined each field by the current IAP response, but some fields are not mentioned
 	// in the following Apple's document;
 	// https://developer.apple.com/library/ios/releasenotes/General/ValidateAppStoreReceipt/Chapters/ReceiptFields.html
 	// If you get other types or fileds from the IAP response, you should use the struct you defined.
 	IAPResponse struct {
-		Status            int     `json:"status"`
-		Environment       string  `json:"environment"`
-		Receipt           Receipt `json:"receipt"`
-		LatestReceiptInfo []InApp `json:"latest_receipt_info"`
-		LatestReceipt     string  `json:"latest_receipt"`
+		Status                   int                  `json:"status"`
+		Environment              string               `json:"environment"`
+		Receipt                  Receipt              `json:"receipt"`
+		LatestReceiptInfo        []InApp              `json:"latest_receipt_info"`
+		LatestReceipt            string               `json:"latest_receipt"`
+		LatestExpiredReceiptInfo InApp                `json:"latest_expired_receipt_info"`
```

Comment:
This field is for iOS 6 style transaction receipt.
But we will not support it because receipt fields are a bit different.
So could you remove this field?